db/event_handler.go

```diff
@@ -57,10 +58,21 @@ func NewWebhook(url string, filterFnString string, timeout *uint64) (*Webhook, e
 	}
 
 	// Initialize transport and client
-	t := http.DefaultTransport.(*http.Transport)
-	transport := *t
-	transport.DisableKeepAlives = false
-	wh.client = &http.Client{Transport: &transport, Timeout: wh.timeout}
+	transport := &http.Transport{
```

Comment:
Could we add a utility function to return a new transport populated with default fields? Then we can reuse it, and modify fields where the new transport is used.

E.g:

```go
	transport := DefaultHTTPTrasnport()
	transport.DisableKeepAlives = false
	wh.client = &http.Client{Transport: transport, Timeout: wh.timeout}
```

```go
// DefaultHTTPTransport returns a new HTTP Transport that copies values from http.DefaultTransport
func DefaultHTTPTransport() http.RoundTripper {

	// This type assertion will panic if http.DefaultTransport ever changes to not be a http.Transport
	// We'll catch this in development/unit testing pretty quickly if it does happen.
	defaultTransport := http.DefaultTransport.(*http.Transport)

	return &http.Transport{
		Proxy:                 defaultTransport.Proxy,
		DialContext:           defaultTransport.DialContext,
		MaxIdleConns:          defaultTransport.MaxIdleConns,
		IdleConnTimeout:       defaultTransport.IdleConnTimeout,
		TLSHandshakeTimeout:   defaultTransport.TLSHandshakeTimeout,
		ExpectContinueTimeout: defaultTransport.ExpectContinueTimeout,
	}
}
```

```go
func TestDefaultHTTPTransport(t *testing.T) {
	assert.NotPanics(t, func() {
		transport := DefaultHTTPTransport()
		assert.NotNil(t, transport, "Returned DefaultHTTPTransport was unexpectedly nil")
	})
}
```